library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
-    * Cataplexy <nowiki> {suggestedTag=Episode-phase, suggestedTag=Finding-significance-to-recording, suggestedTag=Episode-consciousness, suggestedTag=Episode-awareness, suggestedTag=Clinical-EEG-temporal-relationship, suggestedTag=Episode-event-count, suggestedTag=State-episode-start, suggestedTag=Episode-postictal-phase, suggestedTag=Episode-prodrome, suggestedTag=Episode-tongue-biting} [A sudden decrement in muscle tone and loss of deep tendon reflexes, leading to muscle weakness, paralysis, or postural collapse. Cataplexy usually is precipitated by an outburst of emotional expression-notably laughter, anger, or startle. It is one of the tetrad of symptoms of narcolepsy. During cataplexy, respiration and voluntary eye movements are not compromised. Consciousness is preserved.]</nowiki>
-    * Other-episode {requireChild}
+    * Background-activity-special-feature <nowiki> [Special Features. Special features contains scoring options for the background activity of critically ill patients. (Source: Beniczky ea 2017, Section 6.)]</nowiki>
+        ** Continuous-background-activity <nowiki> {suggestedTag=Rhythmic-activity-attribute, suggestedTag=Categorical-location-value, suggestedTag=Sensors} [Source: Beniczky ea 2017, Section 6.]</nowiki>
+        ** Nearly-continuous-background-activity <nowiki> {suggestedTag=Rhythmic-activity-attribute, suggestedTag=Categorical-location-value, suggestedTag=Sensors} [Source: Beniczky ea 2017, Section 6.]</nowiki>
+        ** Discontinuous-background-activity <nowiki> {suggestedTag=Rhythmic-activity-attribute, suggestedTag=Categorical-location-value, suggestedTag=Sensors} [Source: Beniczky ea 2017, Section 6.]</nowiki>
+        ** Background-burst-suppression <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors} [EEG pattern consisting of bursts (activity appearing and disappearing abruptly) interrupted by periods of low amplitude (below 20 microV) and which occurs simultaneously over all head regions. (Source: Beniczky ea 2013, Appendix S2; Beniczky ea 2017, Section 6.)]</nowiki>
+        ** Background-burst-attenuation <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors} [Source: Beniczky ea 2017, Section 6.]</nowiki>
+        ** Background-activity-suppression <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode} [Periods showing activity under 10 microV (referential montage) and interrupting the background (ongoing) activity. (Source: Beniczky ea 2013, Appendix S2; Beniczky ea 2017, Section 6.)]</nowiki>
+        ** Electrocerebral-inactivity <nowiki> [Absence of any ongoing cortical electric activities; in all leads EEG is isoelectric or only contains artifacts. Sensitivity has to be increased up to 2 microV/mm; recording time: at least 30 minutes. (Source: Beniczky ea 2013, Appendix S2; Beniczky ea 2017, Section 6.)]</nowiki>
```

Comment:
I am a little confused about the last part of this.  Is the absence of activity 30 minutes or something else?